app/src/main/res/values/dimens.xml

```diff
@@ -17,6 +17,7 @@
     <dimen name="layout_margin_moderate">10dp</dimen>
     <dimen name="layout_margin_large">16dp</dimen>
     <dimen name="layout_margin_extra_large">32dp</dimen>
+    <dimen name="layout_margin_very_large">56dp</dimen>
```

Comment:
very < extra

Reply:
Can I change extra large 32dp to very large line 19 or take extra extra large for 56dp?